apis/kubedb/v1alpha2/types.go

```diff
@@ -167,3 +170,31 @@ type Age struct {
 	// Populated by Provisioner when authSecret is created or Ops Manager when authSecret is updated.
 	LastUpdateTimestamp metav1.Time `json:"lastUpdateTimestamp,omitempty"`
 }
+
+type Archiver struct {
+	// Pause is used to stop the archiver backup for the database
+	// +optional
+	Pause bool `json:"pause,omitempty"`
+	// Ref is the name and namespace reference to the Archiver CR
+	Ref kmapi.ObjectReference `json:"ref"`
+}
+
+type ArchiverRecovery struct {
+	RecoveryTimestamp metav1.Time `json:"recoveryTimestamp"`
+	// +optional
+	ManifestBackup *ManifestBackupOptions `json:"manifestBackup,omitempty"`
+	// +optional
+	FullBackup *FullBackupOptions `json:"fullBackup,omitempty"`
```

Comment:
We are restoring here. So, `ManifestRestore`, `FullDBRestore` etc are more appropriate word to use